common/lib/xmodule/xmodule/contentstore/content.py

```diff
@@ -290,7 +290,7 @@ def get_canonicalized_asset_path(course_key, path, base_url, excluded_exts, enco
         if encode:
             asset_path = quote_plus(asset_path, '/:+@')
 
-        return urlunparse((None, base_url.encode('utf-8'), asset_path, params, urlencode(updated_query_params), None))
+        return urlunparse(('', base_url.encode('utf-8'), asset_path, params, urlencode(updated_query_params), ''))
```

Comment:
@mraarif Why have you removed None ?

Reply:
it fails in python 3 `urllib` with error `TypeError: Cannot mix str and non-str arguments`

Reply:
I am also facing same issue in my  [21505](https://github.com/edx/edx-platform/pull/21505)